src/main/webapp/app/exercises/programming/shared/code-editor/file-browser/code-editor-file-browser-folder.component.html

```diff
@@ -21,11 +21,11 @@
         <button [disabled]="disableActions" (click)="setCreatingNodeInFolder($event, FileType.FOLDER)" class="btn btn-small">
             <fa-icon [icon]="faFolder" title="{{ 'artemisApp.editor.fileBrowser.createFolder' | artemisTranslate }}"></fa-icon>
         </button>
-        <button [disabled]="disableActions || (isCompressed && item.children && item.children.length)" (click)="setRenamingNode($event)" class="btn btn-small">
+        <button [disabled]="disableActions || (isCompressed && item.value.includes('/'))" (click)="setRenamingNode($event)" class="btn btn-small">
             <fa-icon
                 [icon]="faEdit"
                 title="{{
-                    (!disableActions && isCompressed && item.children && item.children.length
+                    (!disableActions && isCompressed && item.value.includes('/')
```

Comment:
This tooltip is not shown for me

Reply:
I can not reproduce this locally nor on ts1.
What browser are you using? Could you provide a screencast, please?

Reply:
Ah, you're using `title`instead of `ngbTooltip` which has apparently a longer delay until it is shown to the user and I did not wait that long

Reply:
Ahh, ok, nice that it works :)